lib/Sema/CSSimplify.cpp

```diff
@@ -1349,20 +1349,51 @@ ConstraintSystem::matchTupleTypes(TupleType *tuple1, TupleType *tuple2,
   return getTypeMatchSuccess();
 }
 
+// Determine if a function representation conversion is allowed returning
+// 'false' (i.e. no error) if the conversion is valid.
+static bool
+matchFunctionConversionRepresentations(FunctionTypeRepresentation rep1,
```

Comment:
Maybe `isConversionAllowedBetween`? Also comment could be re-phrased into something like "Determine whether conversion is allowed between two function types based on their representations. Return 'false' (i.e. no error if conversion is allowed.".

Reply:
Hey @xedin 
Thanks, I like this naming. I'll adjust :)

Reply:
Adjusted! :)